src/module/system/damage/formula.ts

```diff
@@ -52,10 +52,11 @@ function createDamageFormula(
         const list = typeMap.get(baseEntry.damageType) ?? [];
         typeMap.set(baseEntry.damageType, list);
 
-        if ("terms" in baseEntry) {
+        if (baseEntry.terms) {
             list.push(...baseEntry.terms.map((t) => ({ ...baseEntry, ...t, label: null, critical: null })));
         } else if ((baseEntry.diceNumber && baseEntry.dieSize) || baseEntry.modifier) {
```

Comment:
Won't this still make it so the 0-dice override prevents the base damage from being added to `list`?

Reply:
Yup, but it continues to work just fine. Base only matters for fallbacks, and that's preserved.
![image](https://github.com/foundryvtt/pf2e/assets/1286721/0d7a0e0f-fbe6-408e-8e26-a6694caeba14)